src/core_modules/capture-core/components/Dialogs/ConfirmDialog.component.js

```diff
@@ -3,7 +3,9 @@ import React from 'react';
 import { Modal, ModalTitle, ModalContent, ModalActions, ButtonStrip, Button } from '@dhis2/ui';
 import type { Props } from './confirmDialog.types';
 
-export const ConfirmDialog = ({ open, header, text, confirmText, onConfirm, cancelText, onCancel }: Props) => (
+export const ConfirmDialog = ({
```

Comment:
Hey,
IMHO now that the destructive functionality has been added, the name `ConfirmDialog` is no longer appropriate. Can we change it to something more descriptive? Maybe `DiscardChangesDialog` is a good choice.
Thanks!